### PR TITLE
feat: add a flag to disable monitoring stack for a specific team

### PR DESCRIPTION
**Pull request by srodenhuis**
This PR brings:

- The option to disable the monitoring stack (grafana, prometheus) for a specific team when otomi is running in multi-tenant mode

### CODE REVIEW

File: values/prometheus-operator/prometheus-operator-team.gotmpl [hunk 1]
```diff
@@ -1,5 +1,3 @@
-{{- $teamId := .Release.Labels.team }}
```
Comment (j-zimnowoda) [inline values/prometheus-operator/prometheus-operator-team.gotmpl:1]: Why do you remove fix that was recently merged?

File: helmfile.d/helmfile-60.teams.yaml [hunk 2]
```diff
@@ -130,7 +130,7 @@ releases:
           - low_priority_channel: {{ $team | get "msteams.lowPrio" }}
     {{- end }}
   - name: grafana-dashboards-{{ $teamId }}
-    installed: true
+    installed: {{ $team.monitoringStack.enabled }}
```
Comment (j-zimnowoda) [inline helmfile.d/helmfile-60.teams.yaml:133]: The `$team.monitoringStack.enabled` needs a default value.
Use get `installed: {{ $team | get "monitoringStack.enabled" true }}`

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
Could you take a look at my comments

File: values/prometheus-operator/prometheus-operator-team.gotmpl (hunk 1 above)
Comment (srodenhuis) [reply values/prometheus-operator/prometheus-operator-team.gotmpl]: by accident. Fix has been restored

File: helmfile.d/helmfile-60.teams.yaml (hunk 2 above)
Comment (srodenhuis) [reply helmfile.d/helmfile-60.teams.yaml]: adjusted

**Pull request merged**